public/locales/en/about.json

```diff
@@ -32,6 +32,10 @@
         "part2": "PurpleAir sensors are great for our work because they are affordable and relatively accurate given their cost. That being said, they are not as reliable as some more expensive sensors, which means that sometimes the values they record aren’t sufficient for calculating the AQI. This could be very temporary (a matter of minutes or hours) or might last longer. If a sensor has been down for a long time, it probably means that either the sensor is no longer connected to the internet or that it has encountered a more serious hardware issue.",
         "part3": "If a sensor in your area has been down for a long time, feel free to contact us and we’ll communicate with the sensor host to see if we can resolve the issue."
     },
+    "noSensorInArea": {
+        "heading": "No Sensors In Your Area?",
+        "body": "Some areas of South Gate are better represented on our map than others. Each sensor on the map is physically located somewhere, usually in the backyard of a generous South Gate resident. If there is an area of the city that you would like to see on the map, contact us and let us know! It is especially helpful if you know someone in the area who would be willing to host a sensor in their yard (the sensor requires WiFi and a power outlet)."
```

Comment:
Maybe add the contact info here or direct them to the Get Involved section, but I think a quick repeat of the contact info would be fine too